README.md

```diff
@@ -353,11 +353,10 @@ The argument is a string representing the move, written in some form that's used
 ```python
 stockfish.convert_human_notation_to_sf_notation("e4xd5") # returns "e4d5"
 ```
-Or, say there are two rooks on a5 and b6, and an enemy piece on a6. The following are all valid "human-style" notations for the a5-rook capturing on a6: "Raxa6", "Raa6", "R5xa6", "R5a6".
+Or, say there are two rooks on a5 and b6, and an enemy piece on a6. The following would all be treated as valid "human-style" notations for the a5-rook capturing on a6: "Raxa6", "R5xa6", a5xa6. Note that "Raa6", "R5a6", "a5a6" wouldn't be valid "human-style" notations here, since the move is a capture but there isn't an 'x' indicating this.
```

Comment:
`a5a6` would still go through since that's valid stockfish notation, which is checked first.

Reply:
I tested b3d4 with the fen in the test function though, and it raised a ValueError. Not sure why, since as u said the function checks if it's valid SF notation first.

Reply:
strange indeed. i shall do some testing.

Reply:
my regex is wrong. I added the pawn promotions afterwards and forgot to make it optional

```python
re.match("^(?:[a-h][1-8]){2}[qrnb]$", move.lower())
# should be
regex.match("^(?:[a-h][1-8]){2}[qrnb]?$", move.lower())
```

Reply:
@FalcoGer Sounds good, I've updated the code and it works.
Later on (maybe in this PR or a new one) I might implement a few functions to determine whether the position is checkmate, stalemate, check, etc. Once this is done I should be able to do your suggestion of verifying any '+' or '#' in the argument to this notation function.